dbms/src/Storages/KVStore/MultiRaft/RaftCommands.cpp

```diff
@@ -381,9 +393,9 @@ std::pair<EngineStoreApplyRes, DM::WriteResult> Region::handleWriteRaftCmd(
         case WriteCmdType::Del:
         {
             auto tikv_key = TiKVKey(cmds.keys[i].data, cmds.keys[i].len);
-            if (unlikely(cf == ColumnFamilyType::Write))
+            if (unlikely(cf == ColumnFamilyType::Lock))
             {
-                del_key_count++;
+                lock_del_key_count++;
             }
```

Comment:
Why counting the `lock_del_key` is better than `write_del_key`?

Reply:
`write_del_key` is almost always empty, because everytime a write key is written from a raft normal write command, it will immediately trigger `RegionTable::writeBlockByRegion` which will remove the write key in `RemoveRegionCommitCache`.
So briefly speaking, the "put write cf key" and "del write key" can't be observed in the situation above.
However, there are actually some cases where a "del write key" to its corresponding "put write cf key". One of this case is applying snapshot in which we will leaving some write cf keys if we get trouble in row2col.

Reply:
Either have `write_del_key` or not seems good to me. How do you think?

Reply:
I get it. I'd like to keep this `write_del_key`

Reply:
Ok, we will add it back